docs/tutorials/cargo.md

```diff
@@ -44,37 +51,13 @@ To learn about building ICU4X data, including whether to check in the data blob
 
 [« Fully Working Example »](./crates/buffer)
 
-## Cargo.toml with Baked Provider and build.rs
```

Comment:
Nit: This tutorial example is still valid; you can rename it "Cargo.toml with Custom Baked Data via build.rs"

Reply:
I don't think using baked data via the unstable constructors is something we want to advertise, compiled data is definitely a better solution.

Reply:
Nope you can't. The environment variable needs to be set when `icu_foo_data`'s build.rs runs, which is before the user's build.rs.

Reply:
All build.rs does is output `cargo:rustc-cfg=icu4x_custom_data`; can our build.rs do the same thing and have it affect the dependency? This would be really useful to support; I would like to see a data generation story that works with build.rs.

Reply:
It can't, because it runs after the dependency gets compiled.